test/james.testing.clss.pas

```diff
@@ -44,6 +44,8 @@ interface
   {$ELSE}
     TTestCase = TestFramework.TTestCase;
   {$ENDIF}
+  ITest = James.Testing.ITest;
+  ITestSuite = James.Testing.ITestSuite;
```

Comment:
Putting James.Testing unit bellow will do as well, even though you know how I feel about having to put the units in a specific order! ;)
But it will work, fine with me!